VirtoCommerce.Platform.Web/Scripts/app/dynamicProperties/blades/dynamicProperty-detail.tpl.html

```diff
@@ -102,6 +102,14 @@
                         <input ng-model="data.name" placeholder="{{ 'platform.blades.dynamicProperty-detail.placeholders.display-name' | translate }}" />
                     </div>
                 </div>
+                <div class="form-group">
```

Comment:
More reasonable will be put input tag for description near with `DisplayName` in line 102, isn't it?

Reply:
Here will be a many excess labels tags: name and description for every language. Even we are will be grouping them by language. It will be looking bad in other cases. This is may to stay so or need to reconstruct design of this fully.